st2reactor/st2reactor/garbage_collector/base.py

```diff
@@ -162,6 +164,13 @@ def _perform_garbage_collection(self):
             LOG.debug('Skipping garbage collection for Inquiries since it\'s not '
                       'configured')
 
+        if self._workflow_execution_max_idle > 0:
+            self._purge_orphaned_workflow_executions()
+            eventlet.sleep(self._sleep_delay)
+        else:
+            LOG.debug('Skipping garbage collection for orphaned workflow executions since '
+                      'it\'s not configured')
```

Comment:
Minor style note: it's easier to read single quotes enclosed in double quotes than it is to read escaped single quotes, in my opinion.

Reply:
Thanks for the input.  I copied from another log message in the code above.  I cleaned up all related log messages.